### DISCUSSION TITLE
Login not possible in Eclipse

### DISCUSSION THREAD

**Discussion by tabeaherrmann** [community/community#22539] [New to GitHub]
<p>Hey there,</p>
<p>I want to import a project from an existing git repostitory in Eclipse via URI. The Problem is that my git password and user are not accepted. I double checked username and password multiple times but nothing happens.</p>
<p>Anyone an idea why this happens?</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>You can’t use your password for Git operations, as <a href="https://docs.github.com/en/get-started/getting-started-with-git/why-is-git-always-asking-for-my-password">described in the documentation</a>. You need to <a href="https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token">create a PAT</a> and provide that when asked for your password. Your PAT should have the <em>repo</em> and (if you use Actions) <em>workflow</em> scopes.</p>

**Reply by Muruganandh-scopethinkers** [reply]
Can you explain step by step process

**Reply by BadRothenfelde** [reply]
Worked for me, thanks!

**Reply by SaiWinLart** [reply]
Thanks for the answer, it's work!

**Reply by calebkenney15** [reply]
It worked for me as well, thank you!!

**Comment by tabeaherrmann** [comment]
<p>Thank You so much!!!</p>
